frontend/src/app/search/search.component.html

```diff
@@ -36,7 +36,7 @@
                                 </div>
                             </form>
                         </div>
-                        <ia-download [corpus]="corpus" [queryModel]="queryModel" [resultOverview]="resultOverview" [hasLimitedResults]="hasLimitedResults" [downloadLimit]="user.downloadLimit" [route]="this.router.url" *ngIf="user"></ia-download>
+                        <ia-download [corpus]="corpus" [queryModel]="queryModel" [resultOverview]="resultOverview" [hasLimitedResults]="hasLimitedResults" [downloadLimit]="user? user.downloadLimit: 1000" [route]="this.router.url"></ia-download>
```

Comment:
I'd like to move the fallback value of `downloadLimit` to some centrally located constant if possible.